packages/paste-core/components/skeleton-loader/src/index.tsx

```diff
@@ -1,32 +1,23 @@
 import * as React from 'react';
-import {useSpring, animated} from '@twilio-paste/animation-library';
-import {css, styled} from '@twilio-paste/styling-library';
+import {styled} from '@twilio-paste/styling-library';
 import {Box, safelySpreadBoxProps} from '@twilio-paste/box';
 import type {BoxElementProps} from '@twilio-paste/box';
 import type {LayoutProps, BorderRadiusProps} from '@twilio-paste/style-props';
+import {SkeletonLoaderKeyframes} from './keyframes';
 
-const AnimatedSkeleton = animated(Box);
-
-const StyledAnimatedSkeleton = styled(AnimatedSkeleton)(() =>
-  css({
-    background: `linear-gradient(90deg, transparent, rgba(255, 255, 255, 0.4) 40%, rgba(255, 255, 255, 0.4) 60%, transparent)`,
-    transform: `translateX(-100%) skew(155deg)`,
-  })
-);
-
-const animatedConfig = {
-  loop: {delay: 700, reset: true},
-  from: {translateX: '-100%', skew: '155deg'},
-  // 105% because at 100% with this skew the highlight remains visible
-  // on the bottom right of the loader at the end state.
-  to: {translateX: '105%', skew: '155deg'},
-  config: {
-    mass: 0.1,
-    tension: 80,
-    friction: 50,
-  },
-};
-
+const SkeletonLoaderInner = styled.div({
+  position: 'absolute',
+  top: 0,
+  bottom: 0,
+  left: 0,
+  right: 0,
+  background:
+    'linear-gradient(90deg, transparent, rgba(255, 255, 255, 0.4) 40%, rgba(255, 255, 255, 0.4) 60%, transparent)',
+  animationTimingFunction: 'cubic-bezier(.06,.42,.57,.89)',
```

Comment:
issue (blocking): we need to keep the initial `transform: ...` line because otherwise on computers with prefers-reduced-motion the halted state is with the animation stopped in the center of the skeletonLoader:

Without it:
![image](https://user-images.githubusercontent.com/1592327/165375613-4fa23132-824b-4b3f-a543-de0b67bf32fc.png)

With it: 
![image](https://user-images.githubusercontent.com/1592327/165375814-23d5ae3c-3b60-4808-aa18-57b1664849ef.png)